pandas/core/frame.py

```diff
@@ -3353,38 +3353,23 @@ def assign(self, **kwargs):
 
         Examples
         --------
-        >>> df = pd.DataFrame({'A': range(1, 11), 'B': np.random.randn(10)})
+        >>> df = pd.DataFrame({'A': range(1, 3),
+        ...                    'B': np.arange(-1.0, 2.0, 1.5)})
```

Comment:
Now that there are only few items I think using something like `[1, 2, 3]` would be clearer than `range(1, 3)`. And same for the other parameter.